com.unity.render-pipelines.universal/Shaders/Particles/ParticlesDepthOnlyPass.hlsl

```diff
@@ -0,0 +1,63 @@
+#ifndef UNIVERSAL_PARTICLES_LIT_DEPTH_ONLY_PASS_INCLUDED
+#define UNIVERSAL_PARTICLES_LIT_DEPTH_ONLY_PASS_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
+
+VaryingsDepthOnlyParticle DepthOnlyVertex(AttributesDepthOnlyParticle input)
+{
+    VaryingsDepthOnlyParticle output = (VaryingsDepthOnlyParticle)0;
+    UNITY_SETUP_INSTANCE_ID(input);
+    UNITY_TRANSFER_INSTANCE_ID(input, output);
+    UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(output);
+
+    VertexPositionInputs vertexInput = GetVertexPositionInputs(input.vertex.xyz);
+    output.clipPos = vertexInput.positionCS;
+    output.color = GetParticleColor(input.color);
```

Comment:
can skip color and uv code if _ALPHATEST_ON is disabled

Reply:
Fixed!